tests/specs/Makefile

```diff
@@ -5,7 +5,7 @@ PINSDIR ?= PINS
 SPECS = $(wildcard $(PINSDIR)/*.spec SPECS/*.spec)
 LINKS = $(wildcard $(PINSDIR)/*.pin SPECS/*.lnk)
 MOCK_CONFIGDIR ?= mock
-RPM_EXTRA_DEFINES=--define "_sourcedir %_topdir/SOURCES/%name"
```

Comment:
This line also appears in `xenserver-specs/Makefile`, will it need changing there too?

Reply:
RPM_EXTRA_DEFINES has been superseded by RPM_SOURCEDIR_DEFINE, but now planex/Makefile.rules will set RPM_SOURCEDIR_DEFINE if it isn't set already.

So assuming there isn't some unexpected jiggery-pokery which bypasses that, RPM_SOURCEDIR_DEFINE will only need setting in xenserver-specs/Makefile if it should be different.

I think RPM_EXTRA_DEFINES *should* be removed in xenserver-specs/Makefile for clarity, but has become harmlessly ignored.

Reply:
I think `RPM_EXTRA_DEFINES` has been used in the past to pass random defines into `rpmbuild`. They look to have all gone now, we should remove it to prevent people being misled.

Reply:
Not before merging and deploying this change, however, as everything will break :-)